posthog/utils.py

```diff
@@ -548,10 +549,10 @@ def get_instance_realm() -> str:
     """
     if settings.MULTI_TENANCY:
         return "cloud"
-    elif os.getenv("HELM_INSTALL_INFO", False):
+    elif is_clickhouse_enabled():
```

Comment:
I'm a fan of making this a more accurate check! 

No opinion on the `hosted` vs `hosted-postgres` thing - I think we can just assume that if it isn't hosted-clickhouse it's postgres (what we've been doing anyway).